src/BuildIt.Forms/Samples/BuildIt.Forms.Sample.UI/MainPage.xaml.cs

```diff
@@ -160,5 +160,10 @@ private async void RetrieveAvailableCameras_Clicked(object sender, EventArgs e)
             var cameras = await CameraPreviewControl.RetrieveCamerasAsync();
             AvailableCamerasLabel.Text = string.Join(Environment.NewLine, cameras.Select(c => $"Camera id: {c.Id}, facing: {c.CameraFacing}"));
         }
+
+        private void CameraPreviewControl_MediaFrameArrived(object sender, MediaFrameEventArgs eventArgs)
```

Comment:
Are we doing anything with this?

Reply:
no in the midst of removing that actually